manifests/params.pp

```diff
@@ -2,50 +2,9 @@
 #
 # @api private
 class candlepin::params {
-  $ssl_port = 8443
-  $host = 'localhost'
-
-  $manage_db = true
-  $init_db = true
-  $db_type = 'postgresql'
-  $db_host = 'localhost'
-  $db_user = 'candlepin'
-  $db_name = 'candlepin'
-  $db_port = undef
-  $db_ssl  = false
-  $db_ssl_verify = true
-
   # this comes from keystore
   $db_password = extlib::cache_data('foreman_cache_data', 'candlepin_db_password', extlib::random_password(32))
 
-  $keystore_file = '/etc/candlepin/certs/keystore'
-  $keystore_password = undef
-  $keystore_type = 'PKCS12'
-  $truststore_file = '/etc/candlepin/certs/truststore'
-  $truststore_password = undef
-
-  # where to store output from cpsetup execution
-  $log_dir  = '/var/log/candlepin'
-
-  $crl_file = '/var/lib/candlepin/candlepin-crl.crl'
-
-  $oauth_key = 'candlepin'
-  $oauth_secret = 'candlepin'
-
-  $ca_key = '/etc/candlepin/certs/candlepin-ca.key'
-  $ca_cert = '/etc/candlepin/certs/candlepin-ca.crt'
-  $ca_key_password = undef
-
-  # default to run every 24 hours
-  $expired_pools_schedule = '0 0 0 * * ?'
-
-  # default to disabling the task
-  $certificate_revocation_list_task_schedule='0 0 0 1 1 ?'
-
-  $user_groups = []
-
-  $env_filtering_enabled = true
-
   $ciphers = [
     'SSL_RSA_WITH_3DES_EDE_CBC_SHA',
```

Comment:
I think this is actually an insecure cipher now. Looks like we should update this list as well.